src/coreclr/vm/vars.hpp

```diff
@@ -357,6 +357,7 @@ GVAL_DECL(DWORD,            g_debuggerWordTLSIndex);
 #endif
 GVAL_DECL(DWORD,            g_TlsIndex);
 GVAL_DECL(DWORD,            g_offsetOfCurrentThreadInfo);
+GVAL_DECL(DWORD, g_gcNotificationFlags);
```

Comment:
nit: maintain consistent spacing
```suggestion
GVAL_DECL(DWORD,            g_TlsIndex);
GVAL_DECL(DWORD,            g_offsetOfCurrentThreadInfo);
GVAL_DECL(DWORD,            g_gcNotificationFlags);
```